### PR TITLE
script: ceph-release-notes: minor fixes for split_component

### PR DESCRIPTION
**Pull request by theanalyst**
make labels a set as intersection would fail otherwise, also sort the
results of prefixes before creating the title

Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

### CODE REVIEW

**Comment by theanalyst** [comment]
last set of local changes I had in my repo

**Comment by theanalyst** [comment]
@tchaikov can you review this changeset, this would only affect if --use-tags argument is passed, which used to fail earlier because lists do not have intersection

**Comment by theanalyst** [comment]
jenkins test this please

File: src/script/ceph-release-notes [hunk 1]
```diff
@@ -105,9 +105,9 @@ def split_component(title, gh, number):
         issue_labels = {it['name'] for it in issue['labels']}
         if 'documentation' in issue_labels:
             return 'doc: ' + title
-        item = labels.intersection(issue_labels)
+        item = (labels.union(set(prefixes))).intersection(issue_labels)
```
Comment (tchaikov) [inline src/script/ceph-release-notes]: i am not sure why we want to `union(labels, prefixes)` here? if `issue_labels` are all labels attached to the PR in question, `issue_labels` should be a subset of `labels` as long as the latter is updated. so the return value of `intersection()` should always be `issue_labels`.

File: src/script/ceph-release-notes [hunk 2]
```diff
@@ -38,9 +38,9 @@ from git import Repo
 fixes_re = re.compile(r"Fixes\:? #(\d+)")
 reviewed_by_re = re.compile(r"Rev(.*)By", re.IGNORECASE)
 # labels is the list of relevant labels defined for github.com/ceph/ceph
-labels = ['bluestore', 'build/ops', 'cephfs', 'common', 'core', 'mgr',
+labels = set(['bluestore', 'build/ops', 'cephfs', 'common', 'core', 'mgr',
```
Comment (smithfarm) [inline src/script/ceph-release-notes]: Actually, if I'm not mistaken, the script requires py3 as of e194926301db3e0bb3f9a062be01a70c3c68246e

Comment (tchaikov) [inline src/script/ceph-release-notes]: yes, that's what i meant. py2.7 or up.

File: src/script/ceph-release-notes (hunk 1 above)
Comment (theanalyst) [inline src/script/ceph-release-notes]: ack, this doesn't make sense anymore, my idea was to find the cases where we have a proper title and no labels, but these are already caught before.

File: src/script/ceph-release-notes (hunk 2 above)
Comment (theanalyst) [inline src/script/ceph-release-notes]: will do

File: src/script/ceph-release-notes (hunk 1 above)
Comment (tchaikov) [reply src/script/ceph-release-notes]: @theanalyst ping?

Comment (theanalyst) [reply src/script/ceph-release-notes]: updated, sorry for letting this rot

**Pull request merged**